src/Microsoft.VisualBasic.Core/tests/FinancialTests.cs

```diff
@@ -20,7 +21,7 @@ private static void AreEqual<T>(T expectedOld, T expectedNew, T actual)
             Assert.Equal(expected, actual);
```

Comment:
Could we solve the ARM/Alpine/NetFx problems all together if we just changed this to something like `Assert.Equals(expected.ToString("R8"), actual.ToString("R8"))`?

Reply:
@tannergooding

Reply:
R doesn't respect precision specifies it always returns a roundtrippable string.

Reply:
Others, like 'G', may fix the problem for some and could fail on other results.

Reply:
The old/new (and ARM failures) looked at a glance to be in the 12th decimal position (in scientific notation), or later.  So my off-the-cuff "it's working well enough" was to round high.  I probably meant G8, but what I really meant was "maybe a ToString here can test that it's working to within the tolerance of different FPUs"

Reply:
One thing I thought of doing before was to use the Assert.Equals(a, b, precision) overload and control the precision value based on the platform.

Reply:
>use the Assert.Equals(a, b, precision) overload and control the precision value based on the platform

Thanks @ehasis. I've used your suggestion in https://github.com/dotnet/corefx/pull/39742.